Automattic/wp-calypso
# Hosted Site Migrations: Fix site picker layout width

Related to [#](https://github.com/Automattic/wp-calypso/issues/90754)

## Proposed Changes

* Update site picker width to extra wide layout to better match the old flow.
* Remove blue fill on search icon in site picker.

**Before**

<img width="1508" alt="Screenshot 2024-05-20 at 12 26 36 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/e05be5e2-c56f-47fc-9a17-93c957fa7f44">

**After**

<img width="1512" alt="Screenshot 2024-05-20 at 1 01 13 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/64567df6-b8b5-4525-a2d5-eb336b49c6bc">



## Why are these changes being made?


* the layout looks better with more breathing room

## Testing Instructions



* Switch to this PR
* Navigate to `/setup/hosted-site-migration`
* Note the width of the site picker layout
* Make sure other steps in the flow still use the narrower width

## Pre-merge Checklist
